Traceback (most recent call last):
  File "C:\Users\msayd\Documents\Foundations of Deep Learning\Project\dl-competition\src\main.py", line 243, in <module>
    main(
  File "C:\Users\msayd\Documents\Foundations of Deep Learning\Project\dl-competition\src\main.py", line 79, in main
    image, label = tain_data[i]
NameError: name 'tain_data' is not defined